op-conductor/conductor/service.go

```diff
@@ -140,6 +141,16 @@ func (c *OpConductor) initSequencerControl(ctx context.Context) error {
 	node := sources.NewRollupClient(nc)
 	c.ctrl = client.NewSequencerControl(exec, node)
 
+	enabled, err := retry.Do(ctx, 60, retry.Fixed(5*time.Second), func() (bool, error) {
+		return c.ctrl.ConductorEnabled(ctx)
```

Comment:
This should check if the RPC returns a "method not found" JSON RPC error. If so, then the conductor was updated, but running against an older op-node. In that case, it would fail to rotate the sequencer.
I think it should detect that, log a loud warning that the conductor-check cannot be performed due to outdated op-node, and then allow things to continue without retry (must handle error here, not outside of the retry) and without error.